tests/ui/admin-write-redirect.spec.ts

```
@@ -1,17 +1,25 @@
-const BASE_URL = "http://127.0.0.1:3000";
+function resolveBaseUrl(testInfo: TestInfo) {
+  const baseURL = testInfo.project.use.baseURL;
+  if (!baseURL || typeof baseURL !== "string") {
+    throw new Error("Playwright baseURL is missing");
+  }
+  return baseURL;
+}
 
-test("write compatibility route redirects to admin write", async () => {
-  const response = await fetch(`${BASE_URL}/write`);
+test("write compatibility route redirects to admin write", async ({}, testInfo) => {
```

Biome (2.3.14)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)